linera-core/src/chain_worker/state/temporary_changes.rs

```diff
@@ -219,9 +219,12 @@ where
         if query.request_committees {
             info.requested_committees = Some(chain.execution_state.system.committees.get().clone());
         }
-        if let Some(owner) = query.request_owner_balance {
-            info.requested_owner_balance =
-                chain.execution_state.system.balances.get(&owner).await?;
+        match query.request_owner_balance {
+            owner @ AccountOwner::Application(_) | owner @ AccountOwner::User(_) => {
+                info.requested_owner_balance =
+                    chain.execution_state.system.balances.get(&owner).await?;
+            }
+            _ => {}
```

Comment:
Let's return the chain balance!

Reply:
Good point.

Reply:
Done in [Return chain balance](https://github.com/linera-io/linera-protocol/pull/3560/commits/f04abbc1ac87f1131d16cfb9c78252381a57b1c8)